routes/publication.js

```diff
@@ -0,0 +1,122 @@
+import express from 'express';
+import {getZittingForBesluitenlijst} from '../support/besluit-queries';
```

Comment:
this seems to be missing an import 
`import {getZittingForAgenda} from "../support/agenda-queries";`